test/unit/math/mathOps/src/mathOps.cpp

```diff
@@ -0,0 +1,254 @@
+/** Copyright 2019 Jakob Krude, Benjamin Worpitz
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include "../include/Defines.hpp"
+#include "../include/Buffer.hpp"
+#include "../include/Functor.hpp"
+#include "../include/DataGen.hpp"
```

Comment:
Sorry, I did not notice this in time. I believe it's better to avoid relative paths here, and also in other files added by this PR, and instead do `#include <alpaka/test/unit/math/mathOps/include/Defines.hpp>`. What do you think @jkrude @BenjaminW3 @psychocoderHPC

Reply:
The CMake way would be to add the include folder to the `target_include_directories` of the test target. Then you do not need relative includes anymore.

Reply:
But on the other hand those files are no public includes at all. We should simply move them into the `src` folder.

Reply:
Moving them into `src` is IMO ok. The current relative path requires that the install folder structure  is equal to the structure we have in our source code. We should not assume it.

Reply:
@jkrude when you have time, could you implement this change? Should be not much work, just moving the files and fixing the includes.

Reply:
> @jkrude when you have time, could you implement this change? Should be not much work, just moving the files and fixing the includes.

Already fixed and merged with #986